[13.0][ADD] delivery_price_rule_volumetric_weight: Add Volumetric Weight to delivery price rules (#399)
OCA/delivery-carrier: delivery_price_rule_volumetric_weight/README.rst

   !! This file is generated by oca-gen-addon-readme !!
   !! changes will be overwritten.                   !!
   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
    :target: https://odoo-community.org/page/development-status
    :alt: Beta
.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
    :alt: License: AGPL-3
.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fdelivery--carrier-lightgray.png?logo=github
    :target: https://github.com/OCA/delivery-carrier/tree/13.0/delivery_price_rule_volumetric_weight
    :alt: OCA/delivery-carrier
.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
    :target: https://translation.odoo-community.org/projects/delivery-carrier-13-0/delivery-carrier-13-0-delivery_price_rule_volumetric_weight
    :alt: Translate me on Weblate
.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
    :target: https://runbot.odoo-community.org/runbot/99/13.0
    :alt: Try me on Runbot

|badge1| |badge2| |badge3| |badge4| |badge5| 

> These addon add the possibility to choose volumetric weight in delivery price rules.

Reviewer: ```suggestion
This module adds the possibility of choosing volumetric weight in delivery price rules.
```

**Table of contents**

.. contents::
   :local:

Configuration
=============

To configure this module, you need to:

#. Go to *Sale -> Configuration -> Shipping Methods*.
#. Create a new Shipping method or edit some record and set Based on rule providers.
#. Add new rule and set Volumetric weight in condition and in variable factor.

Usage
=====

To use this module :

#. Go to Sale > Orders > Quotations.
#. Create new order and set some product with volumetric weight.
#. Set some carrier with volumetric weight in rules.

> #. Delivery price have changed according carrier rules.

Reviewer: ```suggestion
#. Delivery price has changed according carrier rules.
```
